src/org/infinity/gui/ResourceChooser.java

```diff
@@ -141,8 +141,11 @@ public String getSelectedType()
   public String getSelectedItem()
   {
     if (lpResources != null) {
-      if (lpResources.getSelectedValue() != null) {
-        return lpResources.getSelectedValue().toString();
+      final ResourceEntry entry = lpResources.getSelectedValue();
+      if (entry != null) {
+        //TODO: toString not designed for such purpoces,
+        // but in all implementations it is equals to getResourceName
+        return entry.toString();
```

Comment:
All implementations implement `toString` as `getResourceName `. May be use there `getResourceName `, because `toString` -- too general method which doesn't have the accurate specification for returning value?